src/main/java/uk/ac/ebi/eva/evaseqcol/model/NameLengthPairEntity.java

```diff
@@ -0,0 +1,25 @@
+package uk.ac.ebi.eva.evaseqcol.model;
```

Comment:
Is there a reason why this is in `model` rather than `entities`?

Reply:
I'm fine with the separation, just wanted to understand the reason - it's true that there's a lot of things in `entities` now!